modules/signing_scriptworker/templates/passwords-mobile.json.erb

```diff
@@ -10,5 +10,8 @@
     ],
     "project:mobile:fenix:releng:signing:cert:release-signing": [
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_fenix_prod_username"]) %>", "<%= scope.function_secret(["autograph_fenix_prod_password"]) %>", ["autograph_fenix"], "autograph"]
+    ],
+    "project:mobile:reference-browser:releng:signing:cert:release-signing": [
```

Comment:
This looks good, presuming we already have the credentials from SecOps. Alternatively, we can maybe feed some dummy values here so that puppet doesn't break because hiera can't find these values, until Wednesday.

Reply:
Maybe I should've posted this in a separate comment, but I [have set up some values in hiera :)](https://github.com/mozilla-releng/build-puppet/pull/303#discussion_r235832740)